README.md

```diff
@@ -91,7 +117,11 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+var doSomething = function(thingToDo){
+  console.log("I need to " + thingToDo);
+}
+
+doSomething("feed the dog");
```

Comment:
Close.  We were expecting you to pass a function, by reference, to doSomething.  doSomething should execute the passed function.